docs_samples/WebSample/README.md

```diff
@@ -1,363 +0,0 @@
-# Deploy a Python (Django) web app with PostgreSQL, Blob Storage, and Managed Identity in Azure
```

Comment:
I believe we still want to include some of this information in `django_sample` README.md.

Reply:
out of date, content changed.